components/textutility.css

```diff
@@ -0,0 +1,57 @@
+:root {
+    --color-gray: rgb(99, 96, 96);
+  }
+.text-h1{
+    font-size: 2rem;
+    padding: 0.5rem;
+}
+.text-h2{
+    font-size: 1.75rem;
+    padding: 0.5rem;
+}
+.text-h3{
+    font-size: 1.5rem;
+    padding: 0.5rem;
+}
+.text-h4{
+    font-size: 1.25rem;
+    padding: 0.5rem;
+}
+.text-h5{
+    font-size: 1rem;
+    padding: 0.5rem;
+}
+.text-h6{
+    font-size: 0.75rem;
+    padding: 0.5rem;
+}
+.text-lg{
+    font-size: 1.5rem;
+    line-height: 1.5;
+    padding: 0.5rem;
+}
+.text-md{
+    font-size: 1.25rem;
+    line-height: 1.5;
+    padding: 0.5rem;
+}
+.text-sm{
+    font-size: 1rem;
+    line-height: 1.5;
+    padding: 0.5rem;
```

Comment:
padding value is same for many classes. Can we club them together?

Reply:
I see only two values being repeated in three classes. Extracting them into a separate class would add another class so I let it be duplicated for now. Any suggestions?

Reply:
Club the class names in css:

.class1 .class2 {
padding: 1rem;
}